AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -204,7 +208,11 @@ struct ChooseSessionTypeView: View {
                                     viewModel.setSearchAndFollow(using: new)
                                 })) {
                                     CreatingSessionFlowRootView {
-                                        SearchView()
+                                        SearchView(isSearchAndFollowFlowActive: .init(get: {
+                                            viewModel.isSearchAndFollowLinkActive
```

Comment:
You are using many time `isSearchAndFollowFlowActive` and here you are passing  `viewModel.isSearchAndFollowLinkActive`. Could you please use one convention throughout the whole concept? (I would suggest `isSearchAndFollowLinkActive`) 😁